VISION.md

```diff
@@ -19,7 +19,7 @@ Instead, MessageKit will provide you with hooks to easily handle your different
 We will strive to support the 3 latest versions of iOS.
 
 - **Objective-C Compatability**: 
-We will not sacrifice functionality or an idiomatic Swift API to support Objective-C.
+We will not sacrifice functionality or an idiomatic Swift API to support Objective-C, but would love to improve Objective-C Compatability where possible.
```

Comment:
```Swift
"Compatability".lowercased()
``` 
please 😃

Reply:
Was just trying to be in line with the header though!